README.md

```diff
@@ -109,10 +257,32 @@ The client API consists of two API sets:
 * iota_client_store_and_broadcast()
 * iota_client_traverse_bundle()
 
-## API Documentation  
+## Examples
 
-TODO
+You can find an example project in the [iota_cclient_example](https://github.com/oopsmonk/iota_cclient_example) directory.
```

Comment:
iota_cclient_example is outdated. maybe [iota_c_templates](https://github.com/oopsmonk/iota_c_templates) fit to a beginner who wanna start a client application.